e2e/rm_test.go

```diff
@@ -271,37 +271,6 @@ func TestRemoveS3PrefixWithoutSlash(t *testing.T) {
 	})
 }
 
-// rm s3://bucket/prefix/
-func TestRemoveS3Prefix(t *testing.T) {
```

Comment:
Why this test is removed? Cant we just update the error message rather than removing it?

Reply:
since we have refactored to make all `rm` validations inside one function and `TestRemoveS3Prefix` test content is already [in the](https://github.com/peak/s5cmd/pull/241/files#diff-fdb5fae95cc63262dac0eb174dc8eb812e177a9c10a18d4c60ebe0a96e7e30e9R35) `TestValidateRMCommand`, I thought removing one might be better.

Reply:
👍